### DISCUSSION TITLE
Failed to register a Japan Amazon account

### DISCUSSION THREAD

**Discussion by samukenull** [audiamus/BookLibConnect#25] [Q&A] [answered]
Hello.

I tried to register my Japanese Amazon account to Book Lib Connect, but when I access the login URL, the login screen does not appear.
When I tried with an Amazon account from another country, the login screen appeared.

I changed all the "amazon.jp" in the login URL to "amazon.co.jp" to try and access the URL, and this time the login screen came up and I was able to get to the "Cannot find page".
However, when I paste the URL into the text box, I get an error message saying "Device registration failed." (I have not registered any other device.)

I've tried it on previous versions and on two different Windows PCs, but it's the same thing. Is there any solution for this?

I'm not very good at English, so sorry for the machine translation.

**Comment by audiamus** [comment]
It should indeed go to  "amazon.co.jp", not "amazon.jp". This may have other side effects, too, but I am not sure. I will fix the domain first, but it will take a few days.

**Reply by audiamus** [reply]
PS: Can you upload the log file and any related files from the tmp directory for my analysis?

**Reply by samukenull** [reply]
Are these correct for the log file? (These files were found in "%userprofile%\AppData\Local\audiamus\BookLibConnect\log".)
I'm sorry, but I don't know which other files are relevant.

[BookLibConnect_2022-02-22_001.log](https://github.com/audiamus/BookLibConnect/files/8115209/BookLibConnect_2022-02-22_001.log)
[BookLibConnect_2022-02-22_002.log](https://github.com/audiamus/BookLibConnect/files/8115210/BookLibConnect_2022-02-22_002.log)
[BookLibConnect_2022-02-22_003.log](https://github.com/audiamus/BookLibConnect/files/8115211/BookLibConnect_2022-02-22_003.log)
[BookLibConnect_2022-02-22_004.log](https://github.com/audiamus/BookLibConnect/files/8115203/BookLibConnect_2022-02-22_004.log)
[BookLibConnect_2022-02-22_005.log](https://github.com/audiamus/BookLibConnect/files/8115204/BookLibConnect_2022-02-22_005.log)
[BookLibConnect_2022-02-22_006.log](https://github.com/audiamus/BookLibConnect/files/8115205/BookLibConnect_2022-02-22_006.log)
[BookLibConnect_2022-02-22_007.log](https://github.com/audiamus/BookLibConnect/files/8115206/BookLibConnect_2022-02-22_007.log)
[BookLibConnect_2022-02-22_008.log](https://github.com/audiamus/BookLibConnect/files/8115207/BookLibConnect_2022-02-22_008.log)
[BookLibConnect_2022-02-22_009.log](https://github.com/audiamus/BookLibConnect/files/8115208/BookLibConnect_2022-02-22_009.log)

**Reply by audiamus** [reply]
Thank you very much for the logs. I guess it will still be the domain. The app uses the domain to register your device. This is done after you paste the login URL into the dialog. And as ".jp" should be ".co.jp", registration currently fails. That's what's shown in log #9, line 50 , for instance: "そのようなホストは不明です。 (api.amazon.jp:443)", i.e. unknown host. Let's wait for 0.5.1, hopefully later this week, to fix this.

BTW, temp. files are found here:  `%userprofile%\AppData\Local\audiamus\BookLibConnect\tmp`. But there won't be much in them at the moment anyway. If 0.5.1 fails as well, we have to take closer look and also run Book Lib Connect with log level 4. But not right now.

**Comment by audiamus** [comment, marked as answer]
I have just released [version 0.5.1 ](https://github.com/audiamus/BookLibConnect/releases/tag/v0.5.1) which should have the correct URL now, I hope.

**Reply by samukenull** [reply]
I just tried it with version 0.5.1, and it successfully registered.

Thank you so much for your quick response!
